### ISSUE TITLE
Create documentation exporter

### ISSUE THREAD

**Issue by blootsvoets** [RADAR-base/RADAR-Schemas#219]
Ideally, the tools will have an exporter that creates a browsable archive of the schemas used. This way, analysts can navigate the documentation of the schemas instead of relying only on a JSON file in the output directory. Specifications can be used to give more comprehensive documentation about the real-time behaviour of a stream, for example:
- caveats about collection rates, accuracy, Android/phone version dependencies
- pseudonymization methods
- causes for missing data
This archive should be hosted by the radar-base platform.

The alternative would be an interactive frontend, however, that would mean that analysts need to keep the RADAR-base instance running even after data collection has finished.

**Comment by afolarin** [comment]
Great idea!
Are you thinking about something like this https://www.openmhealth.org/documentation/#/schema-docs/schema-library but per source rather than data type?

Might be interesting to also provide some basic design checking also on new schema proposals? Perhaps extending the work in `radar-schemas-tools`?

**Comment by blootsvoets** [comment]
Yes, something like that indeed, by extending radar-schemas-tools. Possibly a bit less ambitious than that portal though... We can at least make more accessible the documentation we already have though.
